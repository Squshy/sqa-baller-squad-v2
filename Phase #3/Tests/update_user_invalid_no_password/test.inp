LOGIN
HECKINGCOOLMAN
chimmychonga
CHANGEPASSWORD


logout
exit
