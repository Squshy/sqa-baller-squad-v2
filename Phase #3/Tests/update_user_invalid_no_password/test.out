=========================================================
                    UPDATE PASSWORD
=========================================================
Enter your new password: 
---------------------------------------------------------
You must enter a password.
=========================================================